plugins/services/src/js/components/modals/CreateServiceModalForm.js

```diff
@@ -72,6 +72,34 @@ const CONSTANTLY_UNMUTED_ERRORS = [
   /^volumes\.[0-9]+\./
 ];
 
+function cleanConfig(config) {
+  const { labels = {}, env = {}, environment = {}, ...serviceConfig } = config;
+
+  let newServiceConfig = CreateServiceModalFormUtil.stripEmptyProperties(
```

Comment:
I wonder if we want to keep it. What is there left for cleaning?

Reply:
`portDefinitions` comes to my mind. `containers` in Pods but rather unlikely. `fetch`, `constraints`. There are a couple of array values which we want to clean.

Reply:
I wonder because every time we're trying to touch JSON it breaks someone's process